composer.json

```diff
@@ -10,19 +10,29 @@
 	"require": {
 		"composer/installers": "1.6.0",
 		"ext-openssl": "*",
-		"automattic/jetpack-logo": ">=1.0"
+		"automattic/jetpack-logo": "dev-try/custom-autoloader"
 	},
 	"require-dev": {
 		"dealerdirect/phpcodesniffer-composer-installer": "0.5.0",
 		"wp-coding-standards/wpcs": "2.1.1",
 		"sirbrillig/phpcs-variable-analysis": "2.6.4",
-		"phpcompatibility/phpcompatibility-wp": "2.0.0"
+		"phpcompatibility/phpcompatibility-wp": "2.0.0",
+		"automattic/autoloader": "1.0.0"
 	},
 	"scripts": {
 		"php:compatibility": "composer install && vendor/bin/phpcs -p -s --runtime-set testVersion '5.3-' --standard=PHPCompatibilityWP --ignore=docker,tools,tests,node_modules,vendor --extensions=php",
 		"php:lint": "composer install && vendor/bin/phpcs -p -s",
 		"php:autofix": "composer install && vendor/bin/phpcbf",
-		"php:lint:errors": "composer install && vendor/bin/phpcs -p -s --runtime-set ignore_warnings_on_exit 1"
+		"php:lint:errors": "composer install && vendor/bin/phpcs -p -s --runtime-set ignore_warnings_on_exit 1",
+		"post-install-cmd": [
+			"Jetpack\\Autoloader\\Generator::dump"
+		],
+		"post-update-cmd": [
```

Comment:
Is it necessary to have this again if we have it in `post-autoload-dump`? I was left with the impression that `post-autoload-dump` is ran after each `install` and `update`?

Reply:
I don't know will experiment with it.

Reply:
This was fixed.